admin/database/database.go

```diff
@@ -65,6 +65,7 @@ type DB interface {
 	DeleteOrganizationWhitelistedDomain(ctx context.Context, id string) error
 
 	FindProjects(ctx context.Context, afterName string, limit int) ([]*Project, error)
+	FindProjectNamesByPattern(ctx context.Context, namePattern string) ([]*ProjectNames, error)
```

Comment:
Make it `FindProjectPathsByPattern` and return `[]string, error` instead.

The strings will then have the format `org/project`.

This way, we'll have fewer types here and in the proto, which is nice

Reply:
Also, it should support pagination like `FindUsersByEmailPattern`